modules/tests/integration-tests/basic_construction_tests/basic_construction_test.cpp

```diff
@@ -0,0 +1,200 @@
+#include <gtest/gtest.h>
```

Comment:
directory name integration-tests needs to be changed to integration_tests

Except for that, those tests needs to be separated respectively. it is not necessary to do it now

Reply:
Yeap. changed the name locally to integration_tests and unit_tests. Will land in next commit pack.

In general this basic_test is doomed to be removed as soon as unit_tests will exist.
So I think it is best to remove specific part of test as soon as unit_test is started, until file is empty and can be removed.

Reply:
DONE